Find a functionally equivalent code:com.android.internal.location.GpsNetInitiatedHandler.setNiNotification:COMMENT
Method Modifier: private     internal    
Comment:// Sets the NI notification.

@@ -24,29 +24,30 @@
     // if not to popup dialog immediately, pending intent will open the dialog
     Intent intent = !mPopupImmediately ? getDlgIntent(notif) : new Intent();
     PendingIntent pi = PendingIntent.getBroadcast(mContext, 0, intent, 0);
+    mNiNotification.color = mContext.getResources().getColor(com.android.internal.R.color.system_notification_accent_color);
     mNiNotification.setLatestEventInfo(mContext, title, message, pi);
     notificationManager.notifyAsUser(null, notif.notificationId, mNiNotification, UserHandle.ALL);
 }

